changelog.d/7079.bugfix

```diff
@@ -0,0 +1 @@
+Fix text margin in QR code view when no display name is set
```

Comment:
Wrong comment?

Reply:
now I wonder where I put the correct one 😅